lib/smart_answer_flows/uk-benefits-abroad/questions/is_work_or_sick_pay.govspeak.erb

```diff
@@ -3,8 +3,8 @@
 <% end %>
 
 <% options(
-  "yes": "Yes",
-  "no": "No"
+  "364_days": "364 days",
```

Comment:
(You could put this in one place for consistency, although with a list of 2 items, the benefit is arguable)